scripts/windows/restart-helpers.ps1

```diff
@@ -0,0 +1,64 @@
+# //////////////////////////////////////////////////////////////////////
+# restart-helpers.ps1
+
+function Test-Key([string] $path, [string] $key)
+{
+    return ((Test-Path $path) -and ((Get-Key $path $key) -ne $null))
+}
+
+function Remove-Key([string] $path, [string] $key)
+{
+    Remove-ItemProperty -path $path -name $key
+}
+
+function Set-Key([string] $path, [string] $key, [string] $value)
+{
+    Set-ItemProperty -path $path -name $key -value $value
+}
+
+function Get-Key([string] $path, [string] $key)
+{
+    return (Get-ItemProperty $path).$key
+}
+
+
+$global:restartKey = "Restart-And-Resume"
+function Clear-Any-Restart([string] $key=$global:restartKey)
+{
+    if (Test-Key $global:RegRunKey $key) {
+        Remove-Key $global:RegRunKey $key
+    }
+}
+
+$global:powershell = (Join-Path $env:windir "system32\WindowsPowerShell\v1.0\powershell.exe")
```

Comment:
I like that. And it survives a possible bump to a different version.

Reply:
Nah, they're stuck on `1.0` even though it's actually PowerShell 5.1 here. And PS 6 + will be completely different, since that's PowerShell Core (i.e. the cross-platform one based on .net Core)

… Microsoft naming schemes are opaque.